sdk/dapp-kit/src/hooks/useResolveSuiNSNames.ts

```diff
@@ -1,19 +1,29 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
+import type { ResolvedNameServiceNames } from '@mysten/sui.js/client';
+import type { UseQueryOptions } from '@tanstack/react-query';
+
 import { useSuiClientQuery } from './useSuiClientQuery.js';
 
-export function useResolveSuiNSName(address?: string | null) {
+export function useResolveSuiNSName(
+	address?: string | null,
+	options?: Omit<
+		UseQueryOptions<ResolvedNameServiceNames, Error, ResolvedNameServiceNames, unknown[]>,
```

Comment:
should we exclude `enabled` from here?

Reply:
No, I still want to be be able to disable via options I think